lib/TH/generic/THVectorDispatch.c

```diff
@@ -32,7 +31,7 @@ void THVector_(fill)(real *x, const real c, const ptrdiff_t n) {
   THVector_(fill_DISPATCHPTR)(x, c, n);
 }
 
-
+#ifndef TH_GENERIC_NO_MATH
```

Comment:
just curious, why is fill outside of TH_GENERIC_NO_MATH?  AFAICT it's not called unless TH_GENERIC_HALF_MATH is on anyway?